gordon/interfaces.py

```diff
@@ -0,0 +1,86 @@
+from zope.interface import Attribute
+from zope.interface import Interface
+
+
+class IEventMessage(Interface):
+    id = Attribute('Unique message id')
+    phase = Attribute('Current phase of message')
+    data = Attribute('DNS change data')
+    history_log = Attribute('Log of actions performed on the message')
+
+    def __call__(id, data, phase=None):
+        """Initialize an EventMessage.
```

Comment:
I can't quite understand from the zope docs why one would choose to use `__call__` instead of `__init__` when declaring an interface. Can you tell me why?

Reply:
Thanks for catching this. I got [this part of the zope docs](https://docs.zope.org/zope.interface/README.html#declaring-provided-interfaces) all wrong. After messing around with interfaces in ipython, I updated all the interfaces to use `__init__`.

The zope docs show an example in which the `__init__` method to be different from other methods because it's a class method, which means that its definition should be part of a `IFooFactory` interface instead of `IFoo`.

I don't see any downsides of letting `IFoo` define an `__init__` method so I'll go with this more concise approach.